meetings/README.md

```diff
@@ -0,0 +1,93 @@
+# SAKAI - Online Meetings tool
```

Comment:
Could we review the doc and adjust it to the current situation?

Reply:
done. Pending recordings related permissions